src/test/lambda/explorer/lambdaFunctionNode.test.ts

```diff
@@ -55,4 +62,29 @@ describe('LambdaFunctionNode', function () {
         assert.ok(childNodes)
         assert.strictEqual(childNodes.length, 0, 'Expected node to have no children')
     })
+
+    it('sets appropriate Context Value', function () {
+        const uploadableAndDownloadable: Lambda.FunctionConfiguration = {
+            Runtime: samLambdaImportableRuntimes.first(),
+            PackageType: 'Zip',
+        }
+        const onlyDownloadable: Lambda.FunctionConfiguration = {
+            Runtime: samLambdaImportableRuntimes.first(),
+            PackageType: 'Image',
+        }
+        const onlyUploadable: Lambda.FunctionConfiguration = { Runtime: 'unsupportedRuntime', PackageType: 'Zip' }
+        const neither: Lambda.FunctionConfiguration = { Runtime: 'unsupportedRuntime', PackageType: 'Image' }
+
+        testNode = new LambdaFunctionNode(parentNode, 'someregioncode', uploadableAndDownloadable)
+        assert.strictEqual(testNode.contextValue, CONTEXT_VALUE_LAMBDA_FUNCTION_IMPORTABLE_UPLOADABLE)
+
+        testNode = new LambdaFunctionNode(parentNode, 'someregioncode', onlyDownloadable)
+        assert.strictEqual(testNode.contextValue, CONTEXT_VALUE_LAMBDA_FUNCTION_IMPORTABLE)
+
+        testNode = new LambdaFunctionNode(parentNode, 'someregioncode', onlyUploadable)
+        assert.strictEqual(testNode.contextValue, CONTEXT_VALUE_LAMBDA_FUNCTION_UPLOADABLE)
+
+        testNode = new LambdaFunctionNode(parentNode, 'someregioncode', neither)
+        assert.strictEqual(testNode.contextValue, CONTEXT_VALUE_LAMBDA_FUNCTION)
+    })
```

Comment:
This test could be split into 4 though it wouldn't improve readability that much. I'd update the test name to be more specific though. Something like "sets appropriate contextValue for importing and uploading". Being specific helps to set expectations for the body of test.